### PR TITLE
Completed basic business requirements

### PR DESCRIPTION
**Pull request by andychow94**

### CODE REVIEW

**Review by carolinepeake — DISMISSED** [review]
Good job!  After clicking between related items/outfit and the product detail modules, I received errors for a few bad axios requests. They were requests to the styles API for product 40346. I tried for about 10 minutes to tell where the request was coming from, but couldn't. I did notice you switched the image for the product that didn't have an image to a default image. Thank you for doing that.  Maybe the bad requests were coming from either me or you trying to access the thumbnail images for that product?  I then turned off my module, and clicked arounds yours as much as I could, and again received some errors for bad axios requests. These I could trace to CardList and were when you sent a request for images of a related product, and the related product was the product without images. ('m pretty sure - might want to double check me.). For example, errors were returned when the related/images request was being made with id 40352. Other than that, things looked good!

File: client/src/components/RelatedItems/RelatedList/ComparisonModal.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      features: PropTypes.arrayOf(PropTypes.shape({
+        feature: PropTypes.string,
+        value: PropTypes.string,
+      })),
+      name: PropTypes.string,
+    }),
+  }).isRequired,
+};
+
+const Modal = styled.div`
```
Comment (xgmak94) [inline client/src/components/RelatedItems/RelatedList/ComparisonModal.jsx:74]: Probably should not be an overflow container, should just show all the comparisons at once

File: client/src/components/RelatedItems/RelatedList/CardImage.jsx [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
+      setModal(false);
+    } else {
+      setModal(true);
+    }
   }
 
   return (
-    <div>
-      <ImageCard src={image.imageInfo.results[0].photos[0].thumbnail_url} alt="RelatedProductImage" />
+    <Outline onClick={(e) => { if (modal) { e.stopPropagation(); handleModal(); } }}>
```
Comment (xgmak94) [inline client/src/components/RelatedItems/RelatedList/CardImage.jsx:22]: It is not very obvious how to close the modal, you can instead pass the setModal function to the modal as a prop, and use it within the modal

**Comment by xgmak94** [comment]
Does not currently keep in mind a toggling theme

**Review by xgmak94 — APPROVED** [review]

**Review by ibraheemazam — APPROVED** [review]

**Pull request merged**